src/components/base/Modal/ModalFooter.js

```diff
@@ -1,18 +1,19 @@
 // @flow
 
 import React from 'react'
+import styled from 'styled-components'
 
-import { colors } from 'styles/theme'
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  border-top: 2px solid ${p => p.theme.colors.lightGrey};
+  padding: 20px;
 
-const MODAL_FOOTER_STYLE = {
-  display: 'flex',
-  justifyContent: 'flex-end',
-  borderTop: `2px solid ${colors.lightGrey}`,
-  padding: 20,
-}
+  &:empty {
+    display: none;
+  }
+`
 
-const ModalFooter = ({ children }: { children: any }) => (
-  <div style={MODAL_FOOTER_STYLE}>{children}</div>
-)
+const ModalFooter = ({ children }: { children: any }) => <Wrapper>{children}</Wrapper>
```

Comment:
doesn't that mean that `ModalFooter` === `Wrapper`? :doge:

Reply:
indeed, i didn't even notice. I'll refactor with the (i'm sure) many things coming from @gre's review :trollface: